src/algebra/order/ring.lean

```diff
@@ -383,6 +383,93 @@ protected lemma decidable.one_le_mul_of_one_le_of_one_le [@decidable_rel α (≤
 lemma one_le_mul_of_one_le_of_one_le {a b : α} : 1 ≤ a → 1 ≤ b → (1 : α) ≤ a * b :=
 by classical; exact decidable.one_le_mul_of_one_le_of_one_le
 
+namespace decidable
```

Comment:
What's the point of all this? Nobody ever uses the `decidable.` lemmas and  I feel like they are an historical accident that we shouldn't feed.

Reply:
We have a few users who care about `classical.choice`. Please don't make them angry.

Reply:
As far as I understand, these lemmas are used for natural numbers. Unfortunately, we don't have a list of "big theorems" we want to stay non-classical.

Reply:
I suggest you delete those lemmas, as they will shortly be deduplicated in #16172.